.github/workflows/new-issues.yml

```
@@ -0,0 +1,16 @@
+name: 'Mark new issues with needs-triage label'
+
+on:
+  issues:
+    types:
+      - reopened
+      - opened
```

So, for reusable issues we can just do:

```suggestion
on: [workflow_call]
```

Then the caller can specify:

```yml
on:
  issues:
    types:
      - reopened
      - opened
```

For example: https://github.com/mdn/workflows/blob/main/.github/workflows/idle-issues.yml